autoload/eskk.vim

```diff
@@ -1831,6 +1831,11 @@ function! eskk#enable() abort "{{{
         redrawstatus
         let ret = ''
     endif
+
+    # Specify the "keymap" name
```

Comment:
The comment must be `"` instead of `#`.

Reply:
oh, indeed, forgot switch the "layout" (from ruby/bash/whatever) :)